test/issue-strategies/initiative-test.js

```diff
@@ -1,9 +1,12 @@
 import * as initStrat from '../../src/issue-strategies/initiative.js';
 import DummyJira from '../dummy-jira.js';
 
-let dummyJira = new DummyJira();
-
 describe('Initiative Strategy Apply Tests', () => {
+  let dummyJira;
+
+  beforeEach(function() {
+    dummyJira = new DummyJira();
+  });
```

Comment:
Space between this and the test please. It just makes it clear. Don't be afraid to use arrow functions with the `beforeEach` hooks, either.